pages/screens/Home/index.tsx

```diff
@@ -0,0 +1,29 @@
+import type { NextPage } from 'next';
+import clsx from 'clsx';
+import Head from 'next/head';
+import styles from './styles.module.css';
+
+const Home: NextPage = () => {
+  return (
+    <div className={styles.container} >
+      <Head>
+      <title>Next Challenge | Home</title>
+        <meta name="description" content="Challenge to Next Js generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.main}>
+        <h1 className={clsx('h1')}>Empecemos</h1>
```

Comment:
no need for clsx here

Reply:
Same comment relational to global classes